### PR TITLE
Hms 1

### PR DESCRIPTION
**Pull request by hmscott4**
Issues: #61, #58, #39

### CODE REVIEW

File: AlertManagement/Scripts/Assign-ScomAlert.ps1 [hunk 1]
```diff
@@ -131,26 +131,48 @@ foreach ( $newAlert in $newAlerts )
...
 	foreach ( $searchString in $searchStrings )
 	{
-		$assignmentRule = $config.SelectSingleNode($searchString) |
-		Where-Object -FilterScript { $newAlert.($_.Alert.AlertProperty) -match "$($newAlert.($_.Alert.AlertPropertyMatches))" }
+		$assignmentRule = $config.SelectSingleNode($searchString) | 
+                        Where-Object -FilterScript { $newAlert.($_.Alert.AlertProperty) -match "$($_.Alert.AlertPropertyMatches)" }
 
 		if ( $assignmentRule )
 		{
+            $ruleID = $assignmentRule.ID
```
Comment (randomnote1) [inline AlertManagement/Scripts/Assign-ScomAlert.ps1:148]: Indentation doesn't line up

File: AlertManagement/Scripts/Assign-ScomAlert.ps1 [hunk 2]
```diff
@@ -131,26 +131,48 @@ foreach ( $newAlert in $newAlerts )
...
+	            Write-Debug -Message $message
+            }
 			break
 		}
 	}
 
 	if ( -not $assignedTo )
 	{
 		$assignedTo = $DefaultOwner
+        $ruleID = ""
```
Comment (randomnote1) [inline AlertManagement/Scripts/Assign-ScomAlert.ps1:173]: Use single quotes when setting a string value.

Comment (randomnote1) [inline AlertManagement/Scripts/Assign-ScomAlert.ps1:173]: Indentation doesn't line up

**Review by randomnote1 — CHANGES_REQUESTED** [review]

File: AlertManagement/Scripts/Assign-ScomAlert.ps1 (hunk 2 above)
Comment (hmscott4) [reply AlertManagement/Scripts/Assign-ScomAlert.ps1]: Fixed

File: AlertManagement/Scripts/Assign-ScomAlert.ps1 (hunk 1 above)
Comment (hmscott4) [reply AlertManagement/Scripts/Assign-ScomAlert.ps1]: Fixed

**Pull request merged**